src/ray/protobuf/object_manager.proto

```diff
@@ -50,12 +50,9 @@ message FreeObjectsRequest {
 }
 
 // Reply for request
-message PushReply {
-}
-message PullReply {
-}
-message FreeObjectsReply {
-}
+message PushReply {}
```

Comment:
To make the review easier, could you create a separate PR for this kind of pure format changes?